app/Components/SudokuBoard/SudokuBoard.tsx

```diff
@@ -421,11 +578,70 @@ const SudokuBoard = (props: SudokuBoardProps) => {
   };
 
   /**
-   * Determines the background color of a cell based on the user's settings, the game state, the cell coordinates, and other factors
-   * @param cell The provided cell
-   * @param r The row coordinate of the provided cell
-   * @param c The column coordinate of the provided cell
-   * @returns A hex string that determines the background color of the cell
+   * This function returns an array of 9 strings representing the colors of the
+   * notes for the cell at row r and column c. The colors returned are black
+   * unless the cell is part of a note removal hint and the hint is currently
+   * being displayed. In the case of a note removal hint, the color of the
+   * removed note will be red. The inputs r and c are the row and column of the
+   * cell for which the note colors should be determined.
+   * @param r the row of the cell for which the note colors should be determined
+   * @param c the column of the cell for which the note colors should be
+   * determined
+   */
+  const getCellNotesColor = (r: number, c: number) => {
+    const notesToReturn = [
```

Comment:
Can simplify like this:
```
const notesToReturn = Array(9).fill("black");
```
Even better if can replace 9 with const but not sure where to grab that

Reply:
I pushed up this change